service-food/src/main/kotlin/de/hpi/cloud/food/crawler/openmensa/canteens/Griebnitzsee.kt

```diff
@@ -0,0 +1,38 @@
+package de.hpi.cloud.food.crawler.openmensa.canteens
+
+import de.hpi.cloud.common.entity.Id
+import de.hpi.cloud.food.crawler.openmensa.CanteenData
+import de.hpi.cloud.food.crawler.openmensa.OpenMensaMeal
+import de.hpi.cloud.food.fixtures.labelFixture
+
+object Griebnitzsee : CanteenData(Id("mensaGriebnitzsee"), 62) {
+    private val LABEL_IGNORE_LIST = setOf(
+        "ostern", "ball", "wm"
+    )
+
+    override fun findLabels(meal: OpenMensaMeal): List<String> = meal.notes
+        .map { it.trim().toLowerCase() }
+        .filterNot { it in LABEL_IGNORE_LIST }
+        .mapNotNull { note ->
+            val label = labelFixture.find { label -> label.matches(note) }
+            if (label == null) println("Unknown label \"$note\"")
+            label
+        }
+        .map { it.id.value }
+
+    private fun OpenMensaMeal.categoryMatches(string: String) = category.startsWith(string, ignoreCase = true)
+    override fun findCounter(meal: OpenMensaMeal): String? = when {
+        // TODO: translations
```

Comment:
TODO

Reply:
has to be added in the database later

Reply:
In that case please file a separate issue to track this

Reply:
done in #85